ansible/roles/iroha-docker/tasks/iroha-util.yml

```diff
@@ -37,7 +37,7 @@
 
   - name: Build docker image for iroha-python util
     docker_image:
-      name: "iroha-docker-iroha-python"
+      name: "{{ iroha_util_docker_image }}"
       build:
```

Comment:
Can you add `when` option to this task, to disable it if we use custom `iroha_util_docker_image`

Reply:
Done